test/screenshot/lib/git-repo.js

```diff
@@ -38,12 +38,11 @@ class GitRepo {
   }
 
   /**
-   * @param {string} branch
-   * @param {string=} remote
+   * @param {!Array<string>} args
    * @return {!Promise<void>}
    */
-  async fetch(branch, remote = 'origin') {
-    return this.repo_.fetch([remote, branch]);
+  async fetch(args = []) {
```

Comment:
I'm probably missing something but I've got a question: the signature of this method changed but I don't see any changes to the calls to `fetch`. Is that because the args were never supplied to begin with?

Reply:
The [caller previously passed a value](https://github.com/material-components/material-components-web/pull/2747/files/b75ffbb0528f05583d7542fb612e35e4ccdd53a9#diff-e5acdac7ee2b3f0e9d5340cbfa316fbcL58) for the `branch` param:

```js
fetch(cliArgs.diffBase)
```

But I discovered that it's hard to correctly determine which remote and branch to fetch - it's _much_ simpler and easier to just fetch everything 😛 

AFAICT, it doesn't take too much longer (maybe a second or two?).

Reply:
Cool! I'll take determinism over speed!